.github/workflows/performance.yml

```diff
@@ -86,6 +95,14 @@ jobs:
                       const commit_details = await github.rest.git.getCommit({owner: context.repo.owner, repo: context.repo.repo, commit_sha: context.sha});
                       return parseInt((new Date( commit_details.data.author.date ).getTime() / 1000).toFixed(0))
 
+            - name: Compare performance with custom branches
+              if: github.event_name == 'workflow_dispatch'
+              env:
+                  BRANCHES: ${{ github.event.inputs.branches }}
+                  WP_VERSION: ${{ github.event.inputs.wpversion }}
+              run: |
+                  ./bin/plugin/cli.js perf $(echo $BRANCHES | tr ',' ' ') --tests-branch $GITHUB_SHA --wp-version "$WP_VERSION"
```

Comment:
it looks like the intent is to write branches separated by commas, but since we then split by commas, and since branches can't have spaces, why not pass the list of branches as a space-separated string?

Reply:
I guess in my mind, comma separated strings is like the default way to do arrays. But you're right, I was just dumb here :)

Reply:
not dumb. it just raised the question when I saw the `tr` command